lib/commons/dom/is-visible.js

```diff
@@ -179,6 +193,10 @@ function isVisible(el, screenReader, recursed) {
     return isAreaVisible(el, screenReader, recursed);
   }
 
+  if (nodeName === 'details') {
+    return isDetailsVisible(el);
```

Comment:
I'm a little confused. This seems to suggest to me that the details element is hidden when it isn't open, rather than its content being hidden -- except for the summary element. I'd like to see some tests that show this difference.